classes/class-twentynineteen-walker-comment.php

```diff
@@ -52,10 +52,12 @@ protected function html5_comment( $comment, $depth, $args ) {
 								printf( '<span class="post-author-badge" aria-hidden="true">%s</span>', twentynineteen_get_icon_svg( 'check', 24 ) );
 							}
 
-							/* translators: %s: comment author link */
+							/* translators: %1$s: comment author link */
 							printf(
-								__( '%s <span class="screen-reader-text says">says:</span>' ),
-								sprintf( '<b class="fn">%s</b>', get_comment_author_link( $comment ) )
+								esc_html__( '%1$s %2$ssays:%3$s' ),
```

Comment:
I am not sure if string is easier to read and translate with the many variables.

Reply:
It's definitely less easy to read compared to the prior commit, but it does allow for changing the order to grammatically correct to `Says Sam:`.

With regards to the usage of esc_html__() breaking the html, are you sure? The reason I ask is because the esc_html__() function executes first. So it is only escaping this:`%1$s %2$ssays:%3$s` (or whatever gets passed in from gettext). Then after that, printf() takes the string and swaps in the html for the placeholders.

Reply:
@grappler 

> re:  The escaping is going to cause the HTML to break.

no that won't happen because the call to `esc_html__()` happens **within** the call to `sprintf()`
so `esc_html__()` executes first and its result is passed to `sprintf()`.

the `esc_html__()` is **only** there to protect against HTML injected by the translators,
it will not escape the HTML added by the `sprintf()`

Reply:
> The escaping is going to cause the HTML to break

Sorry I miss read the code. It should not have an effect. 

> It's definitely less easy to read compared to the prior commit,

I meant it was less easier to read then with the HTML tags.